tests/test_source.py

```diff
@@ -283,6 +283,21 @@ def _create_source_config(self, tmpdir, yml_config):
                 'ref': 'b55c00f45ec3dfee0c766cea3d395d6e21cc2e5a',
                 'pkg_managers': ['gomod'],
             }}
+        ), (
+            """\
+            remote_source:
+              repo: https://git.example.com/team/repo.git
+              ref: b55c00f45ec3dfee0c766cea3d395d6e21cc2e5a
+              packages:
+                npm:
+                  - path: client
+                  - path: proxy
+            """,
+            {'remote_source': {
+                'repo': 'https://git.example.com/team/repo.git',
+                'ref': 'b55c00f45ec3dfee0c766cea3d395d6e21cc2e5a',
+                'packages': {'npm': [{'path': 'client'}, {'path': 'proxy'}]},
+            }}
```

Comment:
Can we use the same syntax as for the other entries in the list?

Reply:
Does this look better?

Reply:
I meant the brackets. This is just a nitpick anyway :)
Not blocking on this one.